src/helpers/code-builder.ts

```diff
@@ -0,0 +1,67 @@
+const DEFAULT_INDENTATION_CHARACTER = '';
```

Comment:
The CodeBuilder no longer requiring the `util` module is awesome!

Reply:
Thanks!  Yeah, I didn't actually set out with that goal, but we ended up achieving it by the simple fact of just using regular template strings.